CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## [Unreleased]
 
+## [1.11.0] - 2023-04-27
+
+### Highlights
+
+As of ddtrace 1.11.0, these features are GA and emabled by default:
+
+- CPU Profiling 2.0
+- Remote Configuration
+- Telemetry
+
+For more details, check the release notes.
+
+### Added
+
+* Add remote configuration, enabled by default ([#2674][], [#2678][], [#2686][], [#2687][], [#2688][], [#2689][], [#2696][], [#2705][], [#2731][], [#2732][], [#2733][], [#2739][], [#2756][], [#2769][], [#2771][], [#2773][], [#2789][], [#2805][], [#2794][])
+* AppSec: Add response headers passing to WAF ([#2701][])
+* Tracing: Distributed tracing for Sidekiq ([#2513][])
+* Tracing: Add Roda integration ([#2368][])
+* Profiling: Support disabling endpoint names collection in new profiler ([#2698][])
+* Tracing: Support Sidekiq 7 ([#2810][])
+* Core: Add support for Unix Domain Socket (UDS) configuration via `DD_TRACE_AGENT_URL` ([#2806][])
+* Core: Enable Telemetry by default ([#2762][])
+
+### Changed
+
+* Core: Allow `1` as true value in environment variables ([#2710][])
+* Profiling: Enable CPU Profiling 2.0 by default ([#2702][])
+* Tracing: Improve controller instrumentation and deprecate option `exception_controller` ([#2726][])
+* Tracing: Implement Span Attribute Schema Environment Variable ([#2727][])
+* Tracing: Change default `service_name` values ([#2760][])
```

Comment:
Mention that this change is behind a feature flag. Make sure to update the GitHub release draft as well.